### PR TITLE
LF-4225 Fix for certification document export process

### PR DESCRIPTION
**Pull request by zagran**
**Description**

- Unskipped tests
- Queries updated with `volume` and `weight` ( deprecated `product_quantity`)
- Translations for export updated

Jira link: https://lite-farm.atlassian.net/browse/LF-4225

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [ ] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

**Review by SayakaOno — CHANGES_REQUESTED** [review]
Thank you for taking care of this!

I'm sorry that the requirements weren't clear. Here are the updates needed:
- Remove the unit from the header.
- Add a unit to each cell.

For `volume`, it is stored in l(litter) and for `weight`, it is stored in `kg`. We need to convert these value depending on the user's preferred unit system (metric/imperial):
- volume
  - metric: l
  - imperial: gal
- weight
  - metric: kg
  - imperial: lb

The current conversion logic is implemented here:
https://github.com/LiteFarmOrg/LiteFarm/blob/a0211b826abf8ffbb10f457edf4729791eb6c2d0/packages/api/src/jobs/certification/record_i_generation.js#L24-L32

Before:
Screenshot 2024-07-04 at 3 37 58 PM

After:
Screenshot 2024-07-04 at 3 39 08 PM

I think it would be easier to test if you export the document. Please let us know if you need any guidance. Thank you!

**Comment by zagran** [comment]
@SayakaOno, thanks for your detailed comment. 
I hope I reworked it correctly.

I can't test the export locally for some reason (I probably missed some email configs) - I'll try to find out why

File: packages/api/src/jobs/locales/en/translation.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
       "LISTED_IN_PSL": "Listed in the PSL? (Y/N)",
       "NOTES": "Notes",
       "PRODUCT_NAME": "Product name",
-      "QUANTITY": "Quantity ({{unit}})",
+      "QUANTITY": "Quantity",
```
Comment (SayakaOno) [inline packages/api/src/jobs/locales/en/translation.json:95]: Could you please do the same for the other three languages?

**Review by SayakaOno — COMMENTED** [review]
It's working great, thank you!
It would be great if you cover a few more things 🙏

File: packages/api/src/jobs/locales/en/translation.json (hunk 1 above)
Comment (zagran) [reply packages/api/src/jobs/locales/en/translation.json:95]: Thanks for pointing me at this, fixed

File: packages/api/src/controllers/organicCertifierSurveyController.js [hunk 2]
```diff
@@ -280,8 +281,8 @@ const organicCertifierSurveyController = {
                           t.task_id,
                           p.on_permitted_substances_list
           FROM task t
-                   JOIN soil_amendment_task sat ON sat.task_id = t.task_id
-                   JOIN product p ON p.product_id = sat.product_id
+                   JOIN soil_amendment_task_products satp ON satp.task_id = t.task_id
```
[the remark this answers is not in the document]
Comment (zagran) [reply packages/api/src/controllers/organicCertifierSurveyController.js:284]: https://github.com/LiteFarmOrg/LiteFarm/pull/3288/files#diff-b31c5e334cee002c220af87c2bbb1b8104327e2303d3885513ce61dde7ca3390R304
I hope I understood you correctly, I have added this check check in the query.

[the remark this answers is not in the document]
Comment (SayakaOno) [reply packages/api/src/controllers/organicCertifierSurveyController.js:284]: That's what I meant, thank you!!

**Review by SayakaOno — APPROVED** [review]
Looks perfect, thank you!!

**Comment by Duncan-Brain** [comment]
Sorry it is taking me a bit of time but -- I am getting some unusual results and I am trying to figure out if it is a pre-existing bug or if I am doing something really wrong haha.  Essentially I am seeing is missing and extra records. 

When I run the raw query in my db GUI:

- RecordICrops = returns empty [] 
  - I expected there should be soil amendment product data
  - Downloaded excel sheet is empty
- RecordICleaning = returns with data including soil amendment products
  - I expected there should be just cleaning products
  - My downloaded excel sheet has all data including pest control product data and soil amendment task products (pest control is not present in either raw query -- so weird)

**Review by Duncan-Brain — APPROVED** [review]
@zagran I rolled back my db and it seems like the problems I was having are a pre-existing bug as I am getting the same results. So this PR correctly updates to the new schema! Thank you so much for your patience!

**Pull request merged**